src/Layer/TiledGeometryLayer.js

```diff
@@ -101,6 +102,19 @@ class TiledGeometryLayer extends GeometryLayer {
         this.maxScreenSizeNode = this.sseSubdivisionThreshold * (SIZE_DIAGONAL_TEXTURE * 2);
     }
 
+    get hideSkirt() {
+        return !!this._hideSkirt;
```

Comment:
Could you please initialize `this._hideSkirt` in the constructor. This eases type checking and avoids the double negation here?